src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_denoising_calling.py

```diff
@@ -786,8 +786,7 @@ def __init__(self,
         # the expected number of erroneously mapped reads
         mean_mapping_error_correction_s = eps_mapping * read_depth_s * shared_workspace.average_ploidy_s
 
-        denoised_copy_ratio_st = ((shared_workspace.n_st - mean_mapping_error_correction_s.dimshuffle(0, 'x'))
-                                  / ((1.0 - eps_mapping) * read_depth_s.dimshuffle(0, 'x') * bias_st))
+        denoised_copy_ratio_st = shared_workspace.n_st / (read_depth_s.dimshuffle(0, 'x') * bias_st)
```

Comment:
I feel like this change should have broken a test, but it looks like the only tests test the Python environment.  Should we add tests (maybe not for this PR)?